### PR TITLE
Mailjet (Independent Publisher)

### PR DESCRIPTION
**Pull request by carfup**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

- [X] I attest that the connector works and I verified by deploying and testing all the operations. 
- [X] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [X] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [X] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [X] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [X] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [X] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [X] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md. 

2022_01_07_23-14-17_msedge
2022_01_07_23-15-27_msedge

### CODE REVIEW

File: independent-publisher-connectors/MailJet/apiProperties.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+          "tooltip": "Provide the API Secret",
+          "constraints": {
+            "tabIndex": 3,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#007ee5",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/MailJet/apiProperties.json:31]: @carfup You have agreed that your connector will not have "#007ee5" as iconBrandColor for your connector. Al the IP Connectors must be [set iconBranfColor to "#da3b01"](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiProperties.json).

image

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: independent-publisher-connectors/MailJet/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,2066 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "MailJet",
+    "description": "A complete collection of all publicly available endpoints and methods for the Mailjet API",
+    "version": "1.0",
+    "contact": {
+      "email": "clement@carfup.com",
+      "name": "Cl\u00e9ment OLIVIER",
```
[the remark this answers is not in the document]
Comment (carfup) [reply independent-publisher-connectors/MailJet/apiDefinition.swagger.json]: I removed the "é" which was related to my name. The rest is in full english so no unicode.

File: independent-publisher-connectors/MailJet/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,2066 @@
...
+                        "description": "Status"
+                      },
+                      "Errors": {
+                        "type": "array",
+                        "items": {
+                          "type": "object",
+                          "properties": {
+                            "ErrorIdentifier": {
+                              "type": "string",
+                              "description": "ErrorIdentifier"
```
[the remark this answers is not in the document]
Comment (carfup) [reply independent-publisher-connectors/MailJet/apiDefinition.swagger.json]: I updated the descriptions in order to have proper wording and not "several words in one word"

File: independent-publisher-connectors/MailJet/apiProperties.json (hunk 1 above)
Comment (carfup) [reply independent-publisher-connectors/MailJet/apiProperties.json]: My mistake here. I guess I missed a commit on my end regarding the color...

File: independent-publisher-connectors/MailJet/apiProperties.json [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    "capabilities": [],
+    "publisher": "Clement Olivier",
+	"stackOwner": "MailJet"
```
[the remark this answers is not in the document]
Comment (carfup) [reply independent-publisher-connectors/MailJet/apiProperties.json]: I removed the spaces in front of "stackOwner" to get the proper format

**Comment by sriyen-msft** [comment]
@carfup We are preparing your connector for production deployment and will queue up for the next deployment schedule. Once your connector onboards the next deployment schedule, it starts to deploy your connector in our production environments that typically takes 3 weeks. Please note that the documentation for your connector will not be generated until the deployment process is in the final steps. Thanks.

**Comment by sriyen-msft** [comment]
[[certify-connector]]

**Review by sriyen-msft — APPROVED** [review]

**Comment by sriyen-msft** [comment]
@carfup Apologies for the delay in response. 

Congratulations! The deployment of your connector and documentation page is now complete, We have a short [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program, Thanks.

**Pull request merged**